package.json

```diff
@@ -177,6 +177,7 @@
           "src/components/remotecontrol/remotecontrol.js",
           "src/components/sanatizefilename.js",
           "src/components/scrollManager.js",
+          "src/components/autocast.js",
```

Comment:
I think this makes more sense in scripts for now. Components will hopefully be only UI components at some point.